photon-lib/py/photonlibpy/packet.py

```diff
@@ -22,7 +22,7 @@
 
 
 class Packet:
-    def __init__(self, data: bytes):
+    def __init__(self, data: bytes = b""):
```

Comment:
If we default this to b"" and then encode two packets, does it mutate the same underlying byte array?

Reply:
This is a byte string, which is immutable. Multiple calls will create new objects.